packages/block-editor/src/components/dark-editor-style/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import tinycolor from 'tinycolor2';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+export default function DarkEditorStyle( ref ) {
+	useEffect( () => {
+		const { ownerDocument } = ref.current;
+		const editorStylesWrapper = ownerDocument.getElementsByClassName(
+			'editor-styles-wrapper'
+		);
+		const backgroundColor = window
+			.getComputedStyle( editorStylesWrapper[ 0 ], null )
+			.getPropertyValue( 'background-color' );
+
+		// Calculate the luminosity (0 - 255)
+		const brightness = tinycolor( backgroundColor ).getBrightness();
+
+		if ( brightness > 100 ) {
+			// Is 100 the right number?
+			const body = ownerDocument.getElementsByTagName( 'body' )[ 0 ];
+			// If we only remove it rather than adding it, then if themes don't declare support this will have no impact
```

Comment:
That’s the only part that doesn’t sit all the way.  I guess it would be a surprise to change, but it handling a “non dark mode theme with a dark background” I thought would be handy, and then adding that support wouldn't be necessary anywhere and could be depreciated.

Reply:
Thoughts outlined below suggesting delay of removal of the support make sense to me.